app/models/spree/order_decorator.rb

```diff
@@ -19,6 +19,8 @@ def commit_avatax_invoice(doc_type)
         line_count = 0
         discount = 0
         discount = calculate_order_discounts
+        charge = 0
+        charge = calculate_order_charge
```

Comment:
_charge_ does not seem like a very descriptive variable name. Everything is a charge that is not a credit.
Is this really something more like _adjustment_charges_?